plasmapy/particles/tests/test_atomic.py

```diff
@@ -515,8 +515,7 @@ def test_incorrect_units(self):
             reduced_mass("N", 6e-26 * u.l)
 
     def test_missing_atomic_data(self):
-        with pytest.raises(MissingParticleDataError):
-            reduced_mass("Og", "H")
+        assert reduced_mass("Og", "H") == np.nan
```

Comment:
```suggestion
        assert u.isclose(reduced_mass("Og", "H"), np.nan * u.kg, equal_nan=True)
```
It turns out that `np.nan == np.nan` returns `False` (sigh!), so we have to use the `equal_nan` keyword in `u.isclose` or `np.isclose`.